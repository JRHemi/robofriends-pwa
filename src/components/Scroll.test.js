import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

let wrapper;
beforeEach(() => {
    const MockChild = () => {
        return null;
      };

  wrapper = shallow(<Scroll><MockChild/></Scroll>);
});

it("expects to render Scroll and Child component", () => {
    expect(wrapper).toMatchSnapshot();
  });