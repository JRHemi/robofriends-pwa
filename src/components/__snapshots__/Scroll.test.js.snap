// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Scroll and Child component 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "800px",
      "overflow": "scroll",
    }
  }
>
  <MockChild />
</div>
`;
